presqt/api_v1/utilities/io/write_file.py

```diff
@@ -15,10 +15,14 @@ def write_file(file_path, contents, is_json=False):
     is_json : boolean
         Flag representing if the file should be JSON.
     """
+    # Create Directory
```

Comment:
These two comments might not be necessary. :)
Your code seems pretty clear without them. :thumps-up: